app/src/main/java/tttTDD/Java/Board.java

```diff
@@ -1,44 +1,90 @@
 package tttTDD.Java;
 
-import org.checkerframework.common.value.qual.IntRange;
-
-import java.lang.reflect.Array;
+import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.List;
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
 
 public class Board {
+    private final int DEFAULT_SIZE = 3;
     private final String[] board;
```

Comment:
On the same train of thought as [this comment](https://github.com/rjdeloss/tictactoe-v2-tdd-java/pull/3/files#r874156496), are there any other data types that could more concisely communicate the state of the board?

If so, such a refactor would definitely make sense as it's own PR, just wanted to suggest the idea since this is the first time I'm reviewing code immediately adjacent to it.

Reply:
currently there's an object `GameConfiguration` that gets passed into the game to set the parameters. Eventually there will be an object that holds all the state variables within the `Game`.